# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/webgpu/ComputePassEncoder.cpp
# Commit: f7268832eb50
# Full Hash: f7268832eb50e722776c0eff2ef94d49126a68a0
# Author: Brad Werth <bwerth@mozilla.com>
# Date: 2024-10-17 20:50:15
# Description:
#   Bug 1905383: Make WebGPU encoders not cycle check their subresources. r=webgpu-reviewers,teoxoy
#   
#   The encoder classes already have strong references to the resources they
#   rely upon. That is enough to prevent those resources from being
#   released before the encoder. That's all we need! Including the resources
# ==============================================================================

diff -r 614f990bec46 -r f7268832eb50 dom/webgpu/ComputePassEncoder.cpp
--- a/dom/webgpu/ComputePassEncoder.cpp	Thu Oct 17 13:33:58 2024 +0000
+++ b/dom/webgpu/ComputePassEncoder.cpp	Thu Oct 17 13:43:49 2024 +0000
@@ -13,8 +13,7 @@
 
 namespace mozilla::webgpu {
 
-GPU_IMPL_CYCLE_COLLECTION(ComputePassEncoder, mParent, mUsedBindGroups,
-                          mUsedPipelines)
+GPU_IMPL_CYCLE_COLLECTION(ComputePassEncoder, mParent)
 GPU_IMPL_JS_WRAP(ComputePassEncoder)
 
 void ffiWGPUComputePassDeleter::operator()(ffi::WGPURecordedComputePass* raw) {